.github/CODEOWNERS

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Any committer can add themselves to any  of the path patterns
+# and will subsequently get requested as a reviewer for any PRs
+# that change matching files.
+#
+# This file uses .gitignore syntax with a few exceptions see the
+# documentation about the syntax: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+## Format
+adbc.h @lidavidm
+
+## Implementations
+/c/ @lidavidm @WillAyd
+/csharp/ @CurtHagenlocher @davidhcoe
```

Comment:
You need write access to be a codeowner, so this won't work unfortunately.

Reply:
darn oh well. Should I put you as the codeowner for the csharp? Or leave it commented out for now?

Reply:
...For now, sure. I guess I should go re-learn C♯ at this rate.

Reply:
Strangely enough this still might have pinged me? Unless someone did so in a comment then edited out not sure why I would have been notified by GH

Reply:
@WillAyd I tagged you in a comment, and then deleted that comment after @lidavidm mentioned that it wouldn't work. :)